DuckDuckGo/MainViewController.swift

```diff
@@ -502,8 +502,7 @@ class MainViewController: UIViewController {
     }
     
     func startOnboardingFlowIfNotSeenBefore() {
-        
-        guard ProcessInfo.processInfo.environment["ONBOARDING"] != "false" else {
```

Comment:
@brindy  Just going to validate if we need this, iirc I think it's not actually used properly. If it is needed I think my preference would be to move it into launchOptionsHandler code anyway to turn it into an `or` statement.